examples/nlp/word_language_model/model_def.py

```diff
@@ -0,0 +1,152 @@
+"""
+This example is to show how to use an the PyTorch Word Language Modeling example with Determined.
+The flags and configurations can be found under const.yaml for single GPU training, and distributed.yaml
+for distributed training across 8 GPUs. For more information
+regarding the optional flags view the original script linked below.
+This implementation is based on:
+https://github.com/pytorch/examples/tree/master/word_language_model
+"""
+from typing import Dict, Sequence, Union, Any
+import torch
+import torch.nn as nn
+import time
+from pathlib import Path
+
+from determined.pytorch import (
+    DataLoader,
+    PyTorchTrial,
+    PyTorchTrialContext,
+    LRScheduler,
+)
+import data
+from model import TransformerModel, RNNModel
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+
+class WordLanguageModelPyTorch(PyTorchTrial):
+    def __init__(self, context: PyTorchTrialContext):
+        self.context = context
+        data_config = self.context.get_data_config()
+        hparams = self.context.get_hparams()
+        self.using_bind_mount = data_config.get("use_bind_mount", False)
+        self.use_cache = data_config.get("use_cache", True)
+        self.eval_batch_size = data_config.get("eval_batch_size", 10)
+        self.bind_mount_path = (
+            Path(data_config.get("bind_mount_path")) if self.using_bind_mount else None
+        )
+
+        download_directory = (
+            self.bind_mount_path if self.using_bind_mount else Path("/tmp")
```

Comment:
Why do you need this `Path("/tmp")` here. Can't you directly set else as the `f"data-rank{self.context.distributed.get_rank()}`?

Reply:
Does this mean that the default location for the data should not be `/tmp` but rather `/data`?

Reply:
Yeah I mean this is whatever you prefer. I just want some level of parity between the dtrain path and the single GPU path. Whether it's /data or /tmp, I just want that to be consistent.